docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.
+
+## Introduction to Proof-of-Stake (PoS)
+
+Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to finalize
+blocks on the chain. Consensus is the process of agreeing on something, in this case, the progression
+of the blockchain or how blocks are added to the chain. Consensus consists of two actions:
+
+- `block production`, i.e. the way multiple blocks candidates are produced, and 
+- `block finality`, i.e. the way only one block out of many candidates is selected and added to the canonical chain (see [this](learn-consensus.md/#probabilistic-vs-provable-finality) article for more information about finality).
+
+Proof-of-Work (PoW) and Proof-of-Stake (PoS) are well-known mechanisms used to reach consensus in a secure and trustless way on public blockchains, where there are many participants who do not know each other (and probably never will). In PoW, network security relies on the fact that the miners who are responsible for adding blocks to the chain must compete to solve difficult mathematic puzzles to add blocks - a solution that has been criticized for the wastage of energy. For doing this work, miners are typically rewarded with tokens. In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
+security of the network depends on the amount of capital locked on the chain: the more the capital
+locked, the lower the chance of an attack on the network, as the attacker
+needs to incur a heavy loss to orchestrate a successful attack (more on this later on). The process of
+locking tokens on the chain is called `staking`.
+
+Similar to the miners in PoW networks, PoS networks have `validators`, but they do not have to compete with each other to solve mathematical puzzles.
+They are instead pre-selected to produce the blocks based on the stake backing them.
+Token holders can lock funds on the chain and for doing so, they are getting `staking rewards`. There is
+thus an economic incentive for token holders to become active participants who contribute to the
+security and economic stability of the network. PoS networks in general are therefore more inclusive
+than PoW networks, as participants do not need to have either technical knowledge about
+blockchain technology or experience in running mining equipment. 
+
+PoS ensures that everybody participating in the staking process has
+"skin in the game" and thus can be held accountable. In case of misbehavior, participants in the
+staking process can be punished or `slashed`, and depending on the gravity of the situation, their
+stake can be partly or fully confiscated by the network. It is not in a staker's economic interest to orchestrate an attack and risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the PoS network rewards good behavior and punishes bad behavior.
+
+## Nominated Proof-of-Stake Overview
```

Comment:
Is it worth having a section here to distinguish the difference between nominators, validators and pools? Perhaps it would be good to say Pools is "built" on top of NPoS to provide a very low barrier to entry to staking, without sacrificing Polkadot's strict security model.